woocommerce/woocommerce-admin
# Use clipRule and fillRule props

Part of #4864

Updates use of `fill-rule` and `clip-rule` SVP path properties to be `fillRule` and `clipRule`.

### Detailed test instructions:

- Build branch
- Go to Marketing or Coupon pages
- Make sure Amazon, FB and Gift svgs are visible and there are no console errors

(Note I haven't been able to get the console errors mentioned in the issue to appear (in Chrome or Firefox))
